### ISSUE TITLE
Generate compilable accessors for extension types in the default package

### ISSUE THREAD

**Issue by bamboo** [gradle/kotlin-dsl#807]
This could be fixed by adding import directives to the generated accessors source file, but only for those types in the default package to avoid name conflicts.

**Comment by JLLeitschuh** [comment]
Can you provide an example of this feature? I'm having a hard time picturing what you are talking about.

**Comment by bamboo** [comment]
@JLLeitschuh, sure, the Kotlin DSL currently fails to generate accessors for an extension like the one below, which sits in the default/root package:

```kotlin
// This could be in buildSrc/src/main/kotlin/MyPlugin.kts

// Notice there's no package declaration

// Required imports go here

class MyPlugin : Plugin<Project> {
  override fun apply(project: Project) {
     project.extensions.create("my", MyExtension::class.java)
  }
}

class MyExtension {
  var enabled = false
}
```

**Comment by bamboo** [comment]
Fixed in d709358.
